Overview.src.html

```diff
@@ -835,6 +847,147 @@ <h2 class="no-num no-toc">
+          <span title="startSession">session start</span> algorithm.
+        </p>
+        <section>
+          <h5>
+            Monitoring the list of available presentation displays
+          </h5>
+          <p>
+            When the user agent is to <dfn title=
+            "algorithm-monitor-available">monitor the list of available
+            presentation displays</dfn>, it must run the following steps:
+          </p>
+          <p>
+            While there are alive <span>Availability</span> objects, the user
+            agent should continuously keep track of the <span>list of available
+            presentation displays</span> and repeat the following steps:
+          </p>
+          <ol>
+            <li>
+              <span data-anolis-spec="w3c-html">Queue a task</span> to retrieve
+              the list of currently available <a href=
+              "#presentation-display">presentation displays</a> and let <a id=
+              "newdisplays"><em>newDisplays</em></a> be this list.
+            </li>
+            <li>Wait for the completion of that task.
+            </li>
+            <li>If the <span>list of available presentation displays</span> is
+            empty and <a href="#newdisplays">newDisplays</a> is not empty, then
+            for each alive <code>Availability</code> object:
+              <ol>
+                <li>Set the object's <code>value</code> property to
+                <code>true</code>.
+                </li>
+                <li>
+                  <span data-anolis-spec="w3c-html">Queue a task</span> to
+                  <span data-anolis-spec="w3c-html" title=
+                  "concept-event-fire">fire an event</span> named
+                  <code>change</code> at the object.
+                </li>
+              </ol>
+            </li>
+            <li>If the <span>list of available presentation displays</span> is
+            not empty and <a href="#newdisplays">newDisplays</a> is empty, then
+            for each alive <code>Availability</code> object:
+              <ol>
+                <li>Set the object's <code>value</code> property to
+                <code>false</code>.
+                </li>
+                <li>
+                  <span data-anolis-spec="w3c-html">Queue a task</span> to
+                  <span data-anolis-spec="w3c-html" title=
+                  "concept-event-fire">fire an event</span> named
+                  <code>change</code> at the object.
+                </li>
+              </ol>
+            </li>
+            <li>Set the <span>list of available presentation displays</span> to
+            the value of <a href="#newdisplays">newDisplays</a>.
+            </li>
+          </ol>
+          <p class="note">
+            The mechanism used to monitor <a href=
+            "#presentation-display">presention displays</a> availability is
+            left to the user agent. The user agent may choose to search for
```

Comment:
But if the UA returns an Availability object it's obligated to keep the display status up to date.  Might rephrase as,

The UA may choose to monitor the availability of presentation displays even when there are no Availability objects.